packages/p2p-v2/src/hooks/useAdvertiserStats.ts

```diff
@@ -53,6 +53,9 @@ const useAdvertiserStats = (advertiserId?: string) => {
             /** Checks if the user is already an advertiser */
             isAdvertiser,
 
+            /** Checks if the user is eligible to upgrade their daily limits */
+            isEligibleForLimitUpgrade: Boolean(data?.upgradable_daily_limits) || false,
+
```

Comment:
is the || false condition needed here? won't it default to false if data is undefined or null?

Reply:
🤦 yikes habit